scripts/nci_env_deatest/Test_DEA_Module.sh

```diff
@@ -0,0 +1,188 @@
+                       NCI_SYSTEM_USERNAME NCI/Raijin Login username (ex. sm9911)"
+  echo
+  exit 0
+fi
+
+# Get database name from the config file
+configname=""
+databasename=""
+while IFS=': ' read -r a b; do
+    if [[ "$a" == "["* ]]; then
+       configname=$a
+    fi
+
+    if [[ "$a" == "db_database" && "$configname" == "[datacube]" ]]; then
+       databasename="$b"
+    fi
+done < "$CONFIGFILE"
+
+# Check if output directory exists, else create one
+[ -d "$HOMEDIR/output_files/idx_ingest" ] || mkdir -p "$HOMEDIR"/output_files/idx_ingest
+[ -d "$HOMEDIR/output_files/stats" ] || mkdir -p "$HOMEDIR"/output_files/stats
+[ -d "$HOMEDIR/output_files/fc" ] || mkdir -p "$HOMEDIR"/output_files/fc
+[ -d "$HOMEDIR/output_files/nbconvert" ] || mkdir -p "$HOMEDIR"/output_files/nbconvert
+
+# Function to wait till pbs job is completed
+wait_pbs_job() 
+{
+
+   pbs_qstatus=$(qstat -u "$NCIUSERNAME")
+   while [ ! -z "$pbs_qstatus" ] && [ "$pbs_qstatus" != " " ]
+   do
+        echo 
+        echo "$1 job is executing in Raijin System....."
+        sleep 30s
+        pbs_qstatus=$(qstat -u "$NCIUSERNAME")
+   done
+
+}
+
+echo
+echo "==================================================================="
+echo "|  Env setup_module before index andingest process : $MODULE      "
+echo "==================================================================="
+echo
+# shellcheck source=/dev/null
+source "$HOMEDIR"/python-env/setup_default.sh "$DCCONF"
+
+# Delete previous database (If any)
+dea-test-env -C "$CONFIGFILE" teardown
+sleep 5s
+
+# Create a database file for testing purpose
+createdb -h agdcdev-db.nci.org.au -p 6432 "$databasename"
+
+##################################################################################################
+# Run a test index and ingest on NCI and Raijin system
+##################################################################################################
+echo
+echo "==================================================================="
+echo "|  Test the following on NCI-VDI system:                          |"
+echo "|  1) Indexing  LS7_ETM_NBAR/NBART, LS8_OLITIRS_NBAR/NBART Scenes |"
+echo "|  2) And then ingesting the products to test database            |"
+echo "==================================================================="
+echo
+sh "$HOMEDIR"/dea_testscripts/index_and_ingest.sh "$MODULE" "$CONFIGFILE" "$HOMEDIR" > "$HOMEDIR"/output_files/idx_ingest/Index_Ingest_NCI_PC.log
+
+echo
+echo "==================================================================="
+echo "|  Test the following on Raijin system using PBS:                 |"
+echo "|  1) Indexing  LS7_ETM_NBAR/NBART, LS8_OLITIRS_NBAR/NBART Scenes |"
+echo "|  2) And then ingesting the products to test database            |"
+echo "==================================================================="
+echo
+qsub -v MUT="$MODULE",CONFIG_FILE="$CONFIGFILE",FL_PATH="$HOMEDIR",DC_CONF="$DCCONF" "$HOMEDIR"/dea_testscripts/pbs_script.sh
```

Comment:
What does `FL_PATH` stand for?

Reply:
TEST_BASE would be more appropriate. I will update FL_PATH to TEST_BASE.